docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -290,6 +311,52 @@ type LocalObjectTemplate struct {
   // offered by a provider.
   Ref *corev1.ObjectReference `json:"ref"`
 }
+
+// MachineHealthCheckClass defines a MachineHealthCheck for a group of Machines.
+type MachineHealthCheckClass struct {
+  // UnhealthyConditions contains a list of the conditions that determine
+  // whether a node is considered unhealthy. The conditions are combined in a
+  // logical OR, i.e. if any of the conditions is met, the node is unhealthy.
+  // If not set this field will default to an array of two values:
+  //     - type: Ready
+  //       status: Unknown
+  //       timeout: 300s
+  //     - type: Ready
+  //       status: "False"
+  //       timeout: 300s
```

Comment:
```suggestion
```
Based on that we're deferring the defaulting discussion to a separate issue, I think we should remove this comment for now.

Reply:
I would also remove l.332 and l.347 for now. As long as those fields are only defaulted on the MHC itself I think we shouldn't mention it here. Having the godoc here very strongly applies that this exact fields are defaulted on ClusterClass creation/update.